### ISSUE TITLE
Add minValue, maxValue and MinStep

### ISSUE THREAD

**Issue by NikDevx** [benzman81/homebridge-http-webhooks#131]
Hello @benzman81 
You can add minValue, maxValue and MinStep for HttpWebHookThermostatAccessory.js
Need to add this function to line 35

```
.setProps({
        minValue: 17,
        maxValue: 30,
        minStep: 1
    });
```

**Comment by NikDevx** [comment]
@benzman81 I have already tested and it works :)

**Comment by benzman81** [comment]
@NikDevx Whats the setting for, and why these explicit numbers?

**Comment by NikDevx** [comment]
> @NikDevx Whats the setting for, and why these explicit numbers?

@benzman81 this number will just show you an example

In your plugin, you can add variables and add new parameters to the config.
Like that:

```
    this.service.getCharacteristic(Characteristic.TargetTemperature).on('get', this.getTargetTemperature.bind(this)).on('set', this.setTargetTemperature.bind(this)).setProps({
        minValue: this.minTemp,
        maxValue: this.maxTemp,
        minStep: this.minStep
    });
```

**Comment by NikDevx** [comment]
@benzman81 https://github.com/benzman81/homebridge-http-webhooks/pull/133

**Comment by benzman81** [comment]
Fixed in 0.1.15.
